src/com/oltpbenchmark/api/Procedure.java

```diff
@@ -133,7 +133,11 @@ public final PreparedStatement getPreparedStatementReturnKeys(Connection conn, S
             }
             // Everyone else can use the regular getGeneratedKeys() method
             else if (is != null) {
-                pStmt = conn.prepareStatement(stmt.getSQL(), is);
+                if (getDatabaseType() == DatabaseType.CUBRID) {
```

Comment:
Can you add a comment to explain why you need this special clause?

Reply:
Done.